service/templates/admin.html

```diff
@@ -605,8 +605,8 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<input type="text" ng-model="Settings.scheduler.host_allocator">
 									</md-input-container>
 									<md-input-container class="control" style="width:45%;">
-										<label>Free Host Fraction</label>
-										<input type="number" step="0.01" min="0" max="1"
+										<label>Default Future Host Fraction</label>
+										<input type="number" step=".01" min="-0.1" max="1"
```

Comment:
why allow a negative setting?

Reply:
This allowed me to test the hard-coded default of .4 from the validator, and I thought I might leave it in there. I can remove if you think it's a bad idea.

Reply:
I argue to leave the limit at 0:
- it's not clear to a user that setting a negative number is really setting it to 0.4.
- the code becomes harder to read/maintain: someone looking at it will wonder what we were thinking by allowing up to -0.1. Maybe we had a special meaning in mind.

The other side of the argument would be to allow this so a user can invoke a reasonable default if they have no opinion about the value. One way to accomplish this by making it default when set to 0 (i.e. unset). It would need to be that both the admin setting and distro settings are set to 0, so this is probably more trouble than it's worth.

Reply:
I'm convinced